HW02_A_Graph-Fails.Rmd

```diff
@@ -1,87 +1,104 @@
+
+Second, I wanted to move the legend on top of the graph since there aren't any points there, putting it at approximately the point/ordered pair (5, 40). How do you actually do this? Also, how do you remove the legend title ("class")? Finally, how would you remove the plot legend completely?
+
+ANSWER: To move the legend to the top, set legend.position equal to "top", which I did below. To remove the legend title "class", I added a new theme line that sets legend.title to element_blank(). If I wanted to remove the legend completely, I could set legend.position equal to "none".
+
+```{r}
+mpg %>% 
+ggplot() + 
+  geom_point(mapping = aes(x = displ, y = hwy, color = class), alpha = 1, size = 3) + 
+  theme(legend.direction = "horizontal") + 
+  theme(legend.position = "top") +
+  theme(legend.title = element_blank())
+
+```
+
+### Graph Fail 4
+I wanted just one smoothing line. Just one line, to show the general relationship here. But that's not happening. Instead I'm getting 3 lines, why and fix it please?
+
+ANSWER: I fixed it below. It seems that by defining the color by drv in the ggplot function that the code began to treat each drv group as its own separate data, plotting each's trend. By moving the aes(color = drv) into the geom_point() function, it eliminates this differentiation, meaning the line becomes one that reflects the trend for all the data together, regardless of drv. Additionally, the default smoothening line color is blue, which could get confusing given one of the drv colors is blue, so I changed it to black.
+
+```{r}
+mpg %>% 
+ggplot(mapping = aes(x = displ, y = hwy)) + 
+  geom_point(aes(color = drv)) + 
+  geom_smooth(se = F, color = "black") #se = F makes it so it won't show the error in the line of fit
+```
+
+### Graph Fail 5
+I got tired of the points, so I went to boxplots instead. However, I wanted the boxes to be all one color, but setting the color aesthetic just changed the outline? How can I make the box one color, not just the outline? 
+
+ANSWER: With box plots, you need to manipulate the fill if you want the inside of the box to be a different color. Manipulating color only affects the outline. I added a fill change to make the whole box one color.
+
+Also, the x-axis labels were overlaping, so I rotated them. But now they overlap the bottom of the graph. How can I fix this so axis labels aren't on the graph?
+
+ANSWER: Good call on removing the x axis labels given the legend makes them redundant. You can do that by setting it equal to element_blank(), which I did below.
+
+
+```{r}
+ggplot(data = mpg, mapping = aes(x = manufacturer, y = cty, color = manufacturer, fill = manufacturer)) + 
+  geom_boxplot() + 
+  theme(axis.text.x = element_blank())
```

Comment:
Your solution to the overlapping x-axis labels is perfect! I wish I would have thought of just removing them (LOL). I agree that they're redundant, but if you're in a situation in which you need to keep them and need them to not overlap, you can always adjust their vertical justification as I did with vjust which will shift the text up or down depending on the number you throw in there:
`theme(axis.text.x = element_text(angle = 45, vjust = .65))`